index.html

```diff
@@ -813,6 +823,106 @@ <h3>Classics</h3>
 			   </div>
 		   </section>
 		   <!-- section end -->
+		   <!-- section start --> 
```

Comment:
Suggested that these online resources be added into carousel itself. Do you want me to finish it?

Reply:
yeah please